sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/RegexpExpressionsSuite.scala

```diff
@@ -118,6 +118,295 @@ class RegexpExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     checkLiteralRow("""%SystemDrive%\Users\John""" like _, """\%SystemDrive\%\\Users%""", true)
   }
 
+  test("LIKE Pattern ESCAPE '/'") {
+
```

Comment:
nit: remove a blank.

Reply:
OK.

Reply:
These tests below are super duplicated between each other. IMHO we don't need these over-killing tests and more simpler tests are ok to me here (Basic tests already have been done in https://github.com/apache/spark/pull/25001/files#diff-c36b16be93e0c63a4e3fad37bd904b29R25)
Can you write tests like this?
```
Seq("/", "#", "\"", ...).foreach { escapeChar =>
  test(s"LIKE Pattern ESCAPE '$escapeChar'") {
    // More simpler tests here...
  }
}
```

Reply:
OK. Thanks for your reminder. It is more simpler.